### PR TITLE
Add a warning to Release 0.218 about FULL OUTER JOIN regression

### PR DESCRIPTION
**Pull request by rongrong**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.218.rst [hunk 1]
```diff
@@ -2,6 +2,11 @@
 Release 0.218
 =============
 
+.. warning::
+
+    This release has the potential to produce incorrect results if you have a three way join involving a ``FULL OUTER JOIN`` and one subquery of
```
Comment (nezihyigitbasi) [inline presto-docs/src/main/sphinx/release/release-0.218.rst]: `for three way joins that involve a`

File: presto-docs/src/main/sphinx/release/release-0.218.rst [hunk 2]
```diff
@@ -2,6 +2,11 @@
...
+    the ``FULL OUTER JOIN`` has an `GROUP BY` on join key + some constant. For more details, follow :issue:`12577`.
```
Comment (nezihyigitbasi) [inline presto-docs/src/main/sphinx/release/release-0.218.rst]:
```
a ``GROUP BY``
```

Comment (nezihyigitbasi) [reply presto-docs/src/main/sphinx/release/release-0.218.rst]:
```
on the join key and a constant value. 
```

Comment (nezihyigitbasi) [reply presto-docs/src/main/sphinx/release/release-0.218.rst]:
```
For details, please see :issue:`12577`
```

**Comment by rongrong** [comment]
@nezihyigitbasi comment addressed. Thanks for reviewing.

File: presto-docs/src/main/sphinx/release/release-0.218.rst [hunk 3]
```diff
@@ -2,6 +2,12 @@
 Release 0.218
 =============
 
+.. warning::
+
+    This release has the potential to produce incorrect results for three way joins involving a ``FULL OUTER JOIN`` and one subquery of
+    the ``FULL OUTER JOIN`` has a ``GROUP BY`` on the join key and some expression that could be evaluated to a constant. For details, please
+    refer to :issue:`12577`.
+
 General Changes
```
Comment (nezihyigitbasi) [inline presto-docs/src/main/sphinx/release/release-0.218.rst]: I rephrased this a bit, hope it is still correct.

```
This release may produce incorrect results for three way joins where one of the joins is a ``FULL OUTER JOIN`` and one subquery of the ``FULL OUTER JOIN`` has a ``GROUP BY`` on the join key and an expression that can be evaluated to a constant value. For details, please refer to :issue:`12577`.
```

Comment (rongrong) [reply presto-docs/src/main/sphinx/release/release-0.218.rst]: It's the first join (in order of running). But "one of the joins" is not less inaccurate than "involving".

**Pull request closed without merging**